api/v1alpha1/humiocluster_types.go

```diff
@@ -310,6 +312,35 @@ type HumioNodePoolSpec struct {
 	HumioNodeSpec `json:"spec,omitempty"`
 }
 
+// PodDisruptionBudgetSpec defines the desired pod disruption budget configuration
+type HumioPodDisruptionBudgetSpec struct {
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:Format=int-or-string
+	// +kubebuilder:validation:Immutable
```

Comment:
What is the idea behind making this `Immutable`? I don't see why we would not allow users to change the configs after initially configuring.

Reply:
I have marked those fields immutable to prevent runtime changes that could break the cluster during updates. However, if the operator can safely handle PDB updates through proper requeuing  and rolling update coordination as you previously said, then we could safely remove the flag.

Reply:
I am removing the flag "kubebuilder:validation:Immutable".